lua/super-commit/git/commands.lua

```diff
@@ -1,5 +1,13 @@
 local M = {}
 
+M.map_table = {}
+M.map_table["git-status"] = {winnum=nil, bufnum=nil, command="git status"}
+M.map_table["filelist"] = {winnum=nil, bufnum=nil, command="git diff --name-only --cached"}
+local file_select_suggestion = "To show git diff of files, " ..
+                            "select the file path in the window below, " ..
+                            "and press Enter."
+M.map_table["git-diff-init"] = {winnum=nil, bufnum=nil, command="echo " .. file_select_suggestion}
```

Comment:
This window is also used to show the diff results.
```suggestion
M.map_table["diff"] = {winnum=nil, bufnum=nil, command="echo " .. file_select_suggestion}
```

Reply:
Fixed it in 58468bbc143d815fe46b4a7bdbb207e5ccf89821